public/app/leaderboard.tag

```diff
@@ -30,6 +126,25 @@
       <td class="accounting">{ stat.health_given }</td>
     </tr>
   </table>
+  <div class="leaderboard">
+    <div each={ stat, i in stats } class="leaderboard__hero  hero-stats  { user.getCurrentUserClass(stat.github_login) }">
```

Comment:
What is the reason you used a double-underscore for the class name here and other places?